wrappers/juce/SC3Processor.h

```diff
@@ -73,6 +73,7 @@ class SC3AudioProcessor : public juce::AudioProcessor
     
 
   private:
+    size_t blockpos;
```

Comment:
Yeah I would put the chunk here and call it something like ‘bufferCopyChunk” and make it a static contexpr int or something?

Reply:
I did notice conventions for constants weren't in `ALL_CAPS`. I guess there's no preference?

Reply:
One time in the next week or two I am going to use the CLion refactor tools to go and make them ALL_CAPS like in surge. Just haven't got there yet. So in this case BUFFER_COPY_CHUNK is better indeed.